src/components/workflowEditor/EmptyWorkflow.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { Component } from 'react'
+import { EmptyWorkflowProps, EmptyWorkflowState } from './types'
+import { DialogForm, DialogFormSubmit, ServerErrors, showError } from '@devtron-labs/devtron-fe-common-lib'
+import { createWorkflow } from './service'
+import { toast } from 'react-toastify'
+import error from '../../assets/icons/misc/errorInfo.svg'
+import { FILTER_NAME_REGEX } from '../ApplicationGroup/Constants'
+import {
+    emptyWorkflowMessage,
+    invalidWorkflowNameError,
+    max30CharError,
+    min3CharError,
+    noWorkflowNameError,
+} from './constants'
+
+export default class EmptyWorkflow extends Component<EmptyWorkflowProps, EmptyWorkflowState> {
+    _inputName: HTMLInputElement
```

Comment:
@vivek-devtron Could you please confirm if this needs to converted?

Ref: https://github.com/devtron-labs/dashboard/pull/1483#discussion_r1407302812